.github/workflows/label-new-issues.yml

```diff
@@ -0,0 +1,33 @@
+# Description: This workflow automatically adds a `pending-triage` label to issues that need attention
+#
+# Triggered by: newly opened, re-opened or transferred issues
+
+name: Label new issues
+
+on:
+  issues:
+    types: [opened, reopened, transferred]
+
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+
+    permissions:
+      issues: write
+
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c #v3.3.0 https://github.com/actions/checkout/commit/ac593985615ec2ede58e132d2e21d2b1cbd6127c
+      - name: Add pending-triage label
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ISSUE_NUMBER: ${{ github.event.issue.number }}
+        # sanitize issue number and add label
```

Comment:
Are there docs you can link to for how this sanitization works? It can be hard to read bash commands

Reply:
Could you clarify please? It's essentially the `tr` (removes spaces), `awk` and int commands.. should be common knowledge.